### ISSUE TITLE
bug (Youtube): Crash after some time

### ISSUE THREAD

**Issue by ayushTNM** [inotia00/ReVanced_Extended#2008]
### Type

Error while patching

### Tools used

RVX Manager

### Application

YouTube 19.16.39, rvx patched 4.8.0-dev. 5

### Bug description

After installing and using as normal, youtube crashes (just simply closes) after a certain (seemingly varying) amount of time (around 30 mins to an hour). It can crash at any point, even during a video (not having buffer issues (yet))

### Error logs

```shell
Searching for AndroidRuntime didn't yield any results, so i provided logcat from cutoff point of terminal up untill right after the crash

Included as txt file
```

### Additional context

List of included patches:
{"com.google.android.youtube":["Alternative thumbnails","Ambient mode control","Change player flyout menu toggles","Change shorts repeat state","Change start page","Custom branding icon YouTube","Custom double tap length","Custom package name","Description components","Disable auto audio tracks","Disable auto captions","Disable resuming shorts on startup","Disable splash animation","Enable debug logging","Enable external browser","Enable gradient loading screen","Enable minimized playback","Enable open links directly","Enable tablet mini player","Fullscreen components","GmsCore support","Hide action buttons","Hide ads","Hide comments component","Hide feed components","Hide feed flyout menu","Hide layout components","Hide player button background","Hide player buttons","Hide player flyout menu","Layout switch","MaterialYou","Navigation bar components","Overlay buttons","Player components","Remove viewer discretion dialog","Return YouTube Dislike","Sanitize sharing links","Seekbar components","Settings","Shorts components","Shorts outline button","SponsorBlock","Spoof format stream data","Spoof test client","Toolbar components","Translations","Video playback"]}

### Device Environment

Android 14, Pixel 7

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
Attached logcat does not include crashlog.

**Comment by ayushTNM** [comment]
Hmm, may be the buffer of the terminal (termux). Will update log tomorrow as it is kind of cumbersome to obtain.

**Comment by ayushTNM** [comment]
05-14 13:51:33.127 25548 25548 E AndroidRuntime: FATAL EXCEPTION: main
05-14 13:51:33.127 25548 25548 E AndroidRuntime: Process: app.rvx.android.youtube, PID: 25548
05-14 13:51:33.127 25548 25548 E AndroidRuntime: azqw: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at azsd.a(PG:5)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at aztc.c(PG:14)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at aztc.vS(PG:26)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at bajx.run(PG:124)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at azeh.run(PG:27)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at bapu.run(PG:11)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at bapw.run(PG:34)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:959)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:100)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:232)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:317)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8501)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at app.revanced.integrations.youtube.patches.player.PlayerPatch.setContentDescription(PlayerPatch.java:84)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at znn.K(PG:77)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at znn.v(PG:10)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at znn.b(PG:543)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at zmy.I(PG:2014)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at zmy.d(PG:33)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at aclx.eV(PG:38)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at zot.b(PG:465)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at ztq.h(PG:260)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at ztq.g(PG:61)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at yaz.run(PG:206)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at xkn.e(PG:50)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at xkn.b(PG:7)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at ztq.c(PG:47)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at ztl.c(PG:14)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at abvz.c(PG:423)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at ges.l(PG:139)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at gcq.a(PG:414)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	at aztc.vS(PG:9)
05-14 13:51:33.127 25548 25548 E AndroidRuntime: 	... 12 more

error log

**Comment by inotia00** [comment]
fixed in [revanced-patches-4.8.0-dev.6](https://github.com/inotia00/revanced-patches/releases/tag/v4.8.0-dev.6)

**Comment by inotia00** [comment]
fixed in [revanced-patches-4.8.3](https://github.com/inotia00/revanced-patches/releases/tag/v4.8.3)
